patches/minim/0003-wps-hotplug.sh-Blink-leds-during-wps-discovery.patch

```diff
@@ -0,0 +1,56 @@
+From 29f501e3de4ba4c6410e3bab4de317a6713b069e Mon Sep 17 00:00:00 2001
+From: Brett Mastbergen <brettm@minim.com>
+Date: Wed, 16 Nov 2022 11:18:16 -0500
+Subject: [PATCH] wps-hotplug.sh: Blink leds during wps discovery
+
+SW-2701
+---
+ .../services/hostapd/files/wps-hotplug.sh     | 26 +++++++++++++++++++
+ 1 file changed, 26 insertions(+)
+
+diff --git a/package/network/services/hostapd/files/wps-hotplug.sh b/package/network/services/hostapd/files/wps-hotplug.sh
+index 073bdd1868..b819c923a6 100644
+--- a/package/network/services/hostapd/files/wps-hotplug.sh
++++ b/package/network/services/hostapd/files/wps-hotplug.sh
+@@ -1,5 +1,30 @@
+ #!/bin/sh
+ 
++wps_flash_leds() {
++	local ubusobjs=$1
++	local finished=0
++
++	STATE=$(/sbin/leds.sh led_get_state)
++	[ "$STATE" = "led_off" ] && exit 0
```

Comment:
So, no WPS flash if overall state is LEDs off? I would perceive the LEDs off feature for "because bright LEDs can be annoying all time time", but pushing the WPS button the user is present at the unit for a specific task. Is it possible to flash just the WPS 2.4/5G LEDs even if our state is LEDs off?

Reply:
I assumed if the user toggled off the leds they didn't want any leds.  But yea, we can easily still flash the wifi leds during wps, even in the led_off state, and return to led_off afterwards.  Is that what we want to do?

Reply:
Just wanted to check if it was possible first. I personally think WPS flash is better even if leds are off, but probably want feedback from the product owner. Dave has a meeting scheduled with me for Monday 2:30 to discuss all the LED requirements that aren't in the requirements in Jira. :-P 
Do you want to wait that long? I suppose our other option is check this in now and "fix" it later if necessary